ElectronicObserverTypes/Serialization/FitBonus/FitBonusPerEquipment.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace ElectronicObserverTypes.Serialization.FitBonus
```

Comment:
File scoped namespace, and I think this file is mixing tabs and spaces. Make sure there are tabs everywhere.